litestar/app.py

```diff
@@ -305,6 +305,16 @@ def __init__(
         if debug is None:
             debug = os.getenv("LITESTAR_DEBUG", "0") == "1"
 
+        try:
+            import structlog
+
+            from litestar.logging.config import StructLoggingConfig
+
+            if debug:
+                logging_config = StructLoggingConfig(wrapper_class=structlog.make_filtering_bound_logger(10))
+        except ImportError:
+            logging.basicConfig(level=logging.DEBUG if debug else logging.INFO)
```

Comment:
the logic should not exist here. We have the init logic in the logging module, this is where the update should happen